add command line and sentry button to show links, enable GitLab (#12)
sourcegraph/sentry: README.md

[![codecov](https://codecov.io/gh/sourcegraph/sentry/branch/master/graph/badge.svg)](https://codecov.io/gh/sourcegraph/sourcegraph-sentry)

> # WIP: Sentry extension

Reviewer: Should this still say WIP?

Sentry helps devs track, organize and break down errors more efficiently, facilitating their debug process. We want to make it more convenient for developers to access Sentry's error tracking tools directly from the code that is doing the error handling, code such as `throw new Error(QUERY)`, `console.log(QUERY)`, `console.error(QUERY)` etc..

> In this first version, the Sentry extension will render a `View logs in Sentry` link on each line it detects such error handling code, leading the devs directly to the corresponding Sentry issues stream page. Links will be rendereed when viewing files on [Sourcegraph](https://sourcegraph.com), GitHub and GitLab. 

Reviewer: Should this be in the future tense? also s/rendereed/rendered/

```suggestion
The Sentry extension renders a `View logs in Sentry` next to error throwing statements, linking directly to the corresponding Sentry issues stream page. Links are rendered when viewing files on [Sourcegraph](https://sourcegraph.com), GitHub and GitLab. 
```
